tests/cp2/test_cp2_cbuildcap.py

```diff
@@ -30,6 +30,20 @@
 
 class test_cp2_cbuildcap(BaseBERITestCase):
 
+    # This test uses the claimed decode of the capability
     @attr('capabilities')
     def test_cp2_cbuildcap_1(self):
-        self.assertRegisterEqual(self.MIPS.a0, 1, "CBuildCap did not restore capability correctly")
+        self.assertCapabilitiesEqual(self.MIPS.c1, self.MIPS.c3,
```

Comment:
I would change this to `assert self.MIPS.c1 == self.MIPS.c3, msg`
And the same for the other assertions.
Pytest will print more useful error messages if it can rewrite the full assert.